spec/requests/onboarding/whatsapp_spec.rb

```diff
@@ -7,12 +7,11 @@
     subject { { get: '/onboarding/whats-app' } }
 
     describe 'when no Whatsapp number was configured' do
-      before { allow(Setting).to receive(:whats_app_server_phone_number).and_return('') }
+      before { allow(Setting).to receive(:whats_app_configured?).and_return(false) }
       it { should_not be_routable }
     end
 
     describe 'but when a Whatsapp number was configured' do
-      before { allow(Setting).to receive(:whats_app_server_phone_number).and_return('+49123456789') }
```

Comment:
The tests were more understandable with that mocked values. Now you have to understand that the Setting values fallback to the .env value. Maybe add a comment on that for each of the test just to be nice to everybody?

Reply:
>  Now you have to understand that the Setting values fallback to the .env value. 

I don't know if I understand this sentence correctly, but this is not how the Setting works for `whats_app_server_phone_number`. There are no fallbacks, if there is nothing configured in the .env variables it will return nil. See [Setting model](https://github.com/tactilenews/100eyes/blob/main/app/models/setting.rb#L86):

```rb
field :whats_app_server_phone_number, readonly: true, default: ENV.fetch('WHATS_APP_SERVER_PHONE_NUMBER', nil)
```

With the description:
> but when a Whatsapp number was configured

you need to understand how we configure messengers and need to understand that we have a dedicated `.env` for automated tests.

I don't like adding too many comments in code, to be honest. Of course, we don't have to only follow my preferences. We could maybe add:

```
in .env variables
```

to the description, if you think that would help.